.github/workflows/workflow.yml

```diff
@@ -16,23 +20,30 @@ jobs:
         include:
         - command_description: Build
           command: npm run build
+          needs_lfs: false
           needs_playwright: false
         - command_description: Lint
           command: npm run lint
+          needs_lfs: false
           needs_playwright: false
         - command_description: Less Tests
           command: npm run test:less
+          needs_lfs: false
           needs_playwright: false
         - command_description: Unit Tests
           command: npm run test:unit -- --config web-test-runner.config.ci.mjs
+          needs_lfs: false
           needs_playwright: true
         - command_description: A11y Tests
           command: npm run test:a11y -- --config web-test-runner.config.ci.mjs
+          needs_lfs: false
           needs_playwright: true
         - command_description: Visual Regression Tests
           command: npm run test:visual -- --config web-test-runner.config.ci.mjs
-          needs_playwright: true
+          needs_lfs: true
```

Comment:
Save time on all the other jobs by not pulling down all the images for the visual regressions tests